build.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.7
+#!/usr/bin/env python3
```

Comment:
note that the `#!` shebang is ignored if one calls the script with an explicit python executable